components/IngredientCard/index.tsx

```diff
@@ -154,7 +135,7 @@ const IngredientCards = (props: { ingredients: Ingredient[] }) => {
           lg={3}
           xl={2}
         >
-          <IngredientCard {...quantityData} />
+          <Index {...quantityData} />
```

Comment:
Index is not supper meaningful ;)

```suggestion
          <IngredientCard {...quantityData} />
```